plutus-ledger/src/Ledger/Params.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell   #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+-- | The set of parameters, like protocol parameters and slot configuration.
+module Ledger.Params(
+  Params(..),
+  slotConfigL,
+  protocolParamsL,
+  networkIdL
+) where
+
+import Cardano.Api.Shelley
+import Control.Lens (makeLensesFor)
+import Data.Default (Default (def))
+import Data.Map (fromList)
+import Data.Maybe (fromMaybe)
+import Data.Ratio ((%))
+import Ledger.TimeSlot (SlotConfig)
+import Plutus.V1.Ledger.Api (defaultCostModelParams)
+
+data Params = Params
+  { pSlotConfig     :: SlotConfig
+  , pProtocolParams :: ProtocolParameters
+  , pNetworkId      :: NetworkId
+  }
+  deriving (Eq, Show)
+
+makeLensesFor
+  [ ("pSlotConfig", "slotConfigL")
+  , ("pProtocolParams", "protocolParamsL")
+  , ("pNetworkId", "networkIdL") ]
+  ''Params
+
+instance Default Params where
+  def = Params def def (Testnet $ NetworkMagic 1)
+
+instance Default ProtocolParameters where
```

Comment:
Specify where these default values come from? Is this Alonzo era protocol parameters?